app/views/cancelRegistration/CancelRegistrationDateView.scala.html

```diff
@@ -26,6 +28,13 @@
 
 @(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
+@dateHintText = {
+    @messages("cancelRegistration.cancelRegistrationDate.hintWithin14Days")
+    <p>
```

Comment:
Should perhaps have class="govuk-body" on - not sure though, as hint may take of classes anyway

Reply:
Yes, 
class CancelRegistrationDateFormProviderSpec extends DateBehaviours with SpecBase  {

  val appConfig = application.injector.instanceOf[FrontendAppConfig]
  val form = new CancelRegistrationDateFormProvider(appConfig)()

  ".cancelRegistrationDate" - {

    val validData = datesBetween(
      min = LocalDate.now(),
      max = appConfig.cancelRegistrationDateMaxDaysInFuture
    )

    behave like dateField(form, "cancelRegistrationDate", validData)

    behave like mandatoryDateField(form, "cancelRegistrationDate", "cancelRegistrationDate.error.required.all")
  }
}